app/src/org/commcare/activities/MultipleAppsLimitWarningActivity.java

```diff
@@ -0,0 +1,48 @@
+package org.commcare.activities;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+
+import org.commcare.dalvik.R;
+
+/**
+ * Activity that is shown when a user tries to install more than 2 apps at a time, without having
+ * either superuser privileges or a multiple apps seat enabled on the device.
+ *
+ * @author Aliza Stone (astone@dimagi.com), created 6/9/16.
+ */
+public class MultipleAppsLimitWarningActivity extends CommCareActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        setContentView(R.layout.multiple_apps_limit_view);
+        boolean installAttemptCameFromAppManager = getIntent().getBooleanExtra(AppManagerActivity.KEY_LAUNCH_FROM_MANAGER, false);
+
+        Button toManagerButton = (Button)findViewById(R.id.back_to_manager_button);
+        if (installAttemptCameFromAppManager) {
+            toManagerButton.setText("Back to App Manager");
```

Comment:
Can you load these strings from `string.xml` to prevent the lint warnings/allow for android-level translations?